config/locales/candidate_interface/other_qualification.yml

```
@@ -25,7 +25,7 @@ en:
         change_action: grade
       award_year:
         label: Year qualification was awarded
```

Remove past tense here?

```suggestion
        label: Year qualification awarded
```

---

done 👍